File: programmers/두 정수 사이의 합.cpp
#include <string>
#include <vector>

using namespace std;

long long solution(int a, int b) {
    long long answer = 0;
    if (a > b)
        swap(a, b);
    for (int i = a; i <= b; i++)
        answer += i;

    return answer;
}

File: programmers/level1/두정수사이의합.java
import java.util.*;

class Solution {
  public long solution(int a, int b) {
      long answer = 0;
      int A = a;
      int B = b;
      
      if(a>b) {
         A = b;
         B = a;
      }
      
      for(int i=A; i<=B; i++) {
          answer += i;
      }
      
      return answer;
  }
}